packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -201,13 +203,13 @@ export function applyFieldOverrides(options: ApplyFieldOverrideOptions): DataFra
       }
 
       // Attach data links supplier
-      field.getLinks = getLinksSupplier(
-        newFrame,
-        field,
-        field.state!.scopedVars,
-        context.replaceVariables,
-        options.timeZone
-      );
+      field.getLinks = getLinksSupplier({
+        frame: newFrame,
+        field: field,
+        fieldScopedVars: field.state!.scopedVars,
```

Comment:
Why are we passing this when the function is already getting passed the field

Reply:
I can look into this but my main concern with this was finding out if we could combine two very similar functions without regressions. This is not a change in logic compared to what's there now - the only thing changed was having the params be a single object.